web/src/compose.js

```diff
@@ -699,10 +699,12 @@ export function initialize() {
                 compose_ui.insert_syntax_and_focus(timestr, $target_textarea);
             };
 
+            const date_without_minutes = new Date();
+            date_without_minutes.setMinutes(0, 0);
             flatpickr.show_flatpickr(
                 $(compose_click_target)[0],
                 on_timestamp_selection,
-                new Date(),
+                date_without_minutes,
```

Comment:
Would it make sense for this to call `get_timestamp_for_flatpickr("")`?

Reply:
Hmm, yeah I think it makes more sense to do so.